theme/tchap-otp-login/email/messages/messages_fr.properties

```
@@ -1 +1 @@
-login.code.email.title=Code d''authentification pour r\u00e9server AudioConf
\ No newline at end of file
+login.code.email.title={0} : votre code d''authentification
```

Question/suggestion
Ça marche le genre de chose que je propose ci-dessous ? Ou alors Java n'aime vraiment pas ça du tout ?

```suggestion
login.code.email.title={0} : votre code d’authentification
```

---

hello, je ne pense pas que cela fonctionne et ce n'est pas la convention qu'utilise Keycloak dans ses fichiers.